grails-app/jobs/mama/ng/scheduler/MessageJob.groovy

```diff
@@ -26,7 +28,13 @@ class MessageJob {
         messages.each { Message message ->
             def schedule = message.schedule
 
-            def success = httpRequestService.postText(schedule.endpoint)
+            def body = [
+                "schedule-id": schedule.id,
+                "message-id": message.id,
+                "send-counter": schedule.sendCounter
```

Comment:
The send counter is updated whenever a new message is created. If messages fail there may be  multiple messages for the schedule at the same time? I don't think the scheduler job deletes old messages since we changed it.